### ISSUE TITLE
Highlight already viewed / downloaded images

### ISSUE THREAD

**Issue by GreenGummie** [Bionus/imgbrd-grabber#1964]
When browsing you'll often times see images you've already viewed or downloaded. It would be nice if there was a border around images to signify if you've clicked or downloaded the image, or even just omit them from the result.

**Comment by Bionus** [comment]
Indeed, adding an information on the image if it has already been downloaded, such as a border can be a good idea. Same thing for already viewed images I guess. However, that would require storing the list of all opened images, something Grabber currently does not do.

> or even just omit them from the result

There actually already is a feature to do that. I just added some documentation about it:
https://bionus.github.io/imgbrd-grabber/docs/search.html
(see the "Special “grabber” token" part)

If you just add `-grabber:downloaded` to your post-filters, you won't see already downloaded images in the results anymore.
